components/butterfly/tests/common/mod.rs

```diff
@@ -38,7 +38,9 @@ impl Suitability for NSuitability {
     fn get(&self, _service_group: &str) -> u64 { self.0 }
 }
 
-pub fn start_server(name: &str, ring_key: Option<SymKey>, suitability: u64) -> Server {
+/// # Locking (see locking.md)
+/// * `RumorHeat::inner` (write)
+pub fn start_server_rhw(name: &str, ring_key: Option<SymKey>, suitability: u64) -> Server {
```

Comment:
This function wasn't annotated with `rsw` or `mlw` previously, even though it called `server.start_rsw_mlw`, so we should probably add them for consistency.

Reply:
Ok

Reply:
I didn't bother with annotations like that in test-only code